### PR TITLE
WRP-10546: Added functionality to restore manually set accent/highlight colors

### PR DESCRIPTION
**Pull request by daniel-stoian-lgp**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Changed dynamic color control to CheckboxItem
Controls for accent/hightlight and skinVariant are disabled when dynamicColor is activated.
Added functionality to save manually set accent and highlight color when dynamicColor is activated and restoring them when dynamicColor is deactivated

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRP-10546

### Comments

Enact-DCO-1.0-Signed-off-by: Daniel Stoian [daniel.stoian@lgepartner.com](mailto:daniel.stoian@lgepartner.com)

### CODE REVIEW

File: console/src/views/ThemeSettings.js [hunk 1]
```diff
@@ -227,14 +238,29 @@ ThemeSettingsBase.propTypes = {
 
 ThemeSettingsBase.displayName = 'ThemeSettings';
 
+const ColorCheckboxItem = kind({
```
Comment (stanca-pop-lgp) [inline console/src/views/ThemeSettings.js]: Please move this at the top pf the file where the other custom components are defined. 
Also move DynamicColorSetting below, after SkinVariantsSetting.

File: console/src/views/ThemeSettings.js [hunk 2]
```diff
@@ -182,8 +195,8 @@ const ThemeSettingsBase = (props) => {
 						</Cell>
 						<Cell shrink className={componentCss.spacedItem}>
 							<FormRow align="start space-around" alignLabel="center" className={componentCss.formRow}>
-								<AccentColorSetting label="Accent Color" onClick={onChange}>{swatchPalette}</AccentColorSetting>
-								<HighlightColorSetting label="Highlight Color" onClick={onChange}>{swatchPalette}</HighlightColorSetting>
+								<AccentColorSetting disabled={context.userSettings.dynamicColor} label="Accent Color" onClick={onChange}>{swatchPalette}</AccentColorSetting>
```
Comment (stanca-pop-lgp) [inline console/src/views/ThemeSettings.js]: Maybe we could create a variable called dynamicColorActive for context.userSettings.dynamicColor and use it here and below.

**Review by stanca-pop-lgp — CHANGES_REQUESTED** [review]
Please check my comments.

**Review by stanca-pop-lgp — APPROVED** [review]

**Review by alexandrumorariu — APPROVED** [review]
LGTM

**Pull request merged**